docs/notebooks/microlensing.ipynb

```diff
@@ -323,6 +323,73 @@
     "plt.ylabel(\"Flux\")\n",
```

Comment:
<p>This example feels a little bit artificial: we usually observe μ-lensing in dense fields of Milky Way, so blends are from other field stars, not galaxies. I'd keep the model two component (or hierarchical three): lensed source + (lens + blends), where (lens + blends) term may be represented by a single constant-magnitude tdastro source.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tdastro/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>